docs/self-managed/backup-restore/optimize-backup.md

```diff
@@ -0,0 +1,127 @@
+---
+id: optimize-backup
+title: Backup and restore Optimize data
+description: "How to perform a backup of Optimize data and restore the backup."
+---
+
+Optimize stores its data over multiple indices in Elasticsearch. To ensure data integrity across indices, a backup of Optimize data consists of several Elasticsearch snapshots containing sets of Optimize indices. Each backup is identified by a backup ID. A backup with id `backup1` consists of the following Elasticsearch snapshots:
+
+```
+camunda_optimize_backup1_3.9.0_part_1_of_2
+camunda_optimize_backup1_3.9.0_part_2_of_2
+```
+
+Optimize provides a simple API to trigger a backup and retrieve information about a given backup's state. During backup creation Optimize can continue running. The backed up data can later be restored using the standard Elasticsearch snapshot restore API.
+
+## Prerequisites
+
+The following prerequisites must be set up before using the backup API:
+
+1. A snapshot repository of your choice must be registered with Elasticsearch
+2. The repository name must be specified using the `CAMUNDA_OPTIMIZE_BACKUP_REPOSITORY_NAME` environment variable or by adding it to your Optimize configuration:
+
+```yaml
+backup:
+  repositoryName: <repository name>
+```
+
+## Create backup API
+
+Please note that the backup API can be reached via the `/actuator` management port, by default this is `8092`.  
+The following endpoint can be used to asynchronously trigger the backup process:
+
+```
+POST actuator/backup
+{
+  "backupId": <backupId>
+}
+```
+
+### Response
+
+| Code             | Description                                                                                                                                                                                                                                 |
+| ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 200 OK           | Backup was successfully started, snapshots will be created asynchronously. List of snapshot names is returned in response body (see example below). This list must be persisted together with the backup ID to be able to restore it later. |
```

Comment:
the actual creation is synchronous isn't it? i.e. the second backup is _always_ created after the first?